pkg/apis/configuration/validation/appprotect_test.go

```diff
@@ -1,8 +1,13 @@
 package validation
 
 import (
+	"io"
+	"log/slog"
 	"testing"
 
+	nic_glog "github.com/nginxinc/kubernetes-ingress/internal/logger/glog"
```

Comment:
any specific reason to import separately?

Reply:
I am just going by what Paul has [here](https://github.com/nginxinc/kubernetes-ingress/pull/6585/files#diff-160e0ed58874952875c284ebb762e1893a24adaf9f6b01a9cac9bd16f68a636aR49) to keep consistancy.